.github/workflows/ci-ubuntu.yml

```diff
@@ -6,12 +6,17 @@ jobs:
     strategy:
       matrix:
         perl-version:
-          - '5.30'
+          - "5.30"
```

Comment:
No need to change this.

Reply:
Sorry, didn't mean that. Have to check vscode reformatter for yml files